app/views/curation_concerns/base/_attribute_rows.html.erb

```diff
@@ -1,11 +1,11 @@
-<%= presenter.attribute_to_html(:creator, catalog_search_link: true ) %>
-<%= presenter.attribute_to_html(:contributor, label: 'Contributors', catalog_search_link: true) %>
-<%= presenter.attribute_to_html(:subject, catalog_search_link: true) %>
-<%= presenter.attribute_to_html(:publisher) %>
-<%= presenter.attribute_to_html(:language) %>
-<%= presenter.attribute_to_html(:identifier) %>
-<%= presenter.attribute_to_html(:keyword) %>
-<%= presenter.attribute_to_html(:date_created) %>
-<%= presenter.attribute_to_html(:based_near) %>
-<%= presenter.attribute_to_html(:related_url) %>
-<%= presenter.attribute_to_html(:resource_type) %>
+<%= presenter.attribute_to_html(:creator, render_as: :faceted ) %>
```

Comment:
I switched the creator to `:linked` and get an error. The query in the url bar does not play nice with Sufia.

<img width="1053" alt="screen shot 2016-06-07 at 3 37 15 pm" src="https://cloud.githubusercontent.com/assets/4163828/15872113/c141edee-2cc5-11e6-9dad-932c55cc08f4.png">